android/sound.cpp

```diff
@@ -27,7 +27,7 @@
 
 /* Implementation *************************************************************/
 
-CSound::CSound ( void           (*fpNewProcessCallback) ( CVector<short>& psData, void* arg ),
+CSound::CSound ( void           (*fpNewProcessCallback) ( CVector<float>& psData, void* arg ),
```

Comment:
The "s" in psData stands for "short". This is no longer true. Actually, this variable was incorrect before because "p" stands for pointer but it is not a pointer but a vector. So it should be vfData now since it is float. This applies to a lot of other places too, please check the variable names.

Reply:
Fixed. Ditto for other places I could spot.